testimonials/index.php

```diff
@@ -102,7 +102,7 @@
 </p>
 <p class="center">
   <span class="testimonial">"We are delighted to be in a position to distribute Aboriginal language keyboard solutions to our clients. The local Saanich tribe will be the first to benefit ... You can be sure that your invention will be a wonderful enabling tool for a host of new Indigenous communities via their connections to First Peoples' Cultural Foundation."</span>
-  <h4 class="caption">Peter Brand - <a href="www.fpcf.ca" target="blank">First Peoples' Cultural Foundation</a>, British Columbia</h4>
+  <h4 class="caption">Peter Brand - <a href="https://www.fpcf.ca/" target="_blank">First Peoples' Cultural Foundation</a>, British Columbia</h4>
```

Comment:
Maybe their site is http only? On my Win 10 / Chrome
https://www.fpcf.ca/ times out
http://fpcf.ca/ gets served up